pdc/apps/compose/migrations/0012_auto_20160615_0611.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('compose', '0011_remove_duplicate_pathtypes'),
+    ]
+
```

Comment:
Could combine 0011 into this migration? Is it because re-index and change content can't in one transaction so can't work?